subsys/mgmt/hawkbit/hawkbit.c

```diff
@@ -1482,12 +1490,27 @@ static void autohandler(struct k_work *work)
 	case HAWKBIT_PROBE_IN_PROGRESS:
 		LOG_INF("hawkBit is already running");
 		break;
+
+	default:
+		LOG_ERR("Invalid response: %d", response);
+		break;
 	}
 
 	if (k_work_delayable_from_work(work) == &hawkbit_work_handle) {
 		k_work_reschedule(&hawkbit_work_handle, K_SECONDS(poll_sleep));
 	}
 }
+
+enum hawkbit_response hawkbit_autohandler_wait(uint32_t events, k_timeout_t timeout)
```

Comment:
Do you expect anyone would use anything other than `__UINT32_MAX__` for `events`?

Reply:
I can think of some people only using `BIT(HAWKBIT_NO_UPDATE)`